.github/workflows/run_functional_product_tests.yaml

```diff
@@ -41,18 +41,23 @@ jobs:
         shell: pwsh # -Password flag requires PS 6+
         env:
           PfxPassword: ${{ secrets.NIGHTLY_TEST_BUILD_PW }}
-      - name: Setup for Scuba
-        run: ./SetUp.ps1
-      - name: Install Selenium
-        run: Install-Module -Name Selenium
-      - name: Update Selenium
-        run: ./Testing/Functional/SmokeTest/UpdateSelenium.ps1
+
+      - name: Install Selenium & Update Wed Drivers
+        run: |
+          Install-Module -Name Selenium -Scope CurrentUser -Force
+          Import-Module -Name Selenium
+          ./Testing/Functional/SmokeTest/UpdateSelenium.ps1
+          Remove-Module -Name Selenium
       - name: Test Products
         # The testing params are stored as a string in a CSV-like format.
         # Pipes separate products, and commas separate key/value pairs.
         # key1=value1,key2=value2|key3=value3,key4=value4...
         # This job parses the keys and values and converts them to a hashtable.
         run: |
+          Import-Module -Name .\PowerShell\ScubaGear\ScubaGear.psd1
+          Install-OPA
+          # Workaround for Selenium. Loading psm1 instead of psd1
+          Import-Module -Name (Get-Module -Name Selenium -ListAvailable).Path -Force
```

Comment:
These should be in a previous setup step, not the step that is parsing the params and calling the product tests.

Reply:
Moved to previous, setup, step